tests/builder/custom_params_test.py

```diff
@@ -120,31 +123,37 @@ def test_set_fsSelection_flags(self):
         self.set_custom_params()
         self.assertEqual(self.ufo.info.openTypeOS2Selection, [7, 8])
 
-    def test_underlinePosition(self):
-        self.master.customParameters["underlinePosition"] = -2
-        self.set_custom_params()
-        self.assertEqual(self.ufo.info.postscriptUnderlinePosition, -2)
-
-        # self.master.customParameters['underlinePosition'] = 1
-        for param in self.master.customParameters:
-            if param.name == "underlinePosition":
-                param.value = 1
-                break
-        self.set_custom_params()
-        self.assertEqual(self.ufo.info.postscriptUnderlinePosition, 1)
-
-    def test_underlineThickness(self):
-        self.master.customParameters["underlineThickness"] = 100
-        self.set_custom_params()
-        self.assertEqual(self.ufo.info.postscriptUnderlineThickness, 100)
+    def test_integer_parameters(self):
+        """Test casting glyphsapp customParameters whose values are just
+        integers into ufo equivalents."""
+        integer_params = [
+            "underlinePosition",
+            "underlineThickness",
+            "underlinePosition",
+            "underlineThickness",
```

Comment:
You're testing twice the same parameters.
Also the strikeoutSize and Position are missing here

Reply:
Good catch. Thank you.